include/spm/evt_mario.h

```diff
@@ -14,6 +14,12 @@ EVT_DECLARE_USER_FUNC(evt_mario_flag8_onoff, 2)
 UNKNOWN_FUNCTION(func_800ef53c);
 UNKNOWN_FUNCTION(evt_mario_cont_onoff);
 
+EVT_DECLARE_USER_FUNC(evt_mario_pos_change, 3)
+EVT_DECLARE_USER_FUNC(evt_mario_fairy_reset, 0)
+EVT_DECLARE_USER_FUNC(evt_mario_direction_face, 2)
+EVT_DECLARE_USER_FUNC(evt_mario_direction_reset, 0)
```

Comment:
Function order again
Unnamed verisons of `evt_mario_direction_face` and `evt_mario_direction_reset` don't seem to have been removed